src/SQLitePCLRaw.lib.e_sqlcipher.ios/SQLitePCLRaw.lib.e_sqlcipher.ios.targets

```diff
@@ -0,0 +1,16 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(RuntimeIdentifier)'=='ios-arm64' or '$(ComputedPlatform)'=='iPhone'">
```

Comment:
`ComputedPlatform` is set both for legacy `Xamarin.iOS` and new `.NET 7` platforms. I kept the `RuntimeIdentifier` checks for extra clarity since we may want to support plain `net6.0` eventually.